mkdocs/docs/experiments/categories/pods/pod-http-latency.md

```diff
@@ -0,0 +1,363 @@
+## Introduction
+
+- It injects http request latency on the specified container by starting proxy server and then redirecting the traffic to the proxy server.
+- It can test the application's resilience to lossy/flaky http requests
+
+!!! tip "Scenario: Corrupt the network packets of target pod"    
+    ![Pod Network Corruption](../../images/network-chaos.png)
```

Comment:
Since the Chaos Variables are different for this experiment, can you add a new image? For the sake of uniformity, use draw.io

Reply:
Have updated the image now